### DISCUSSION TITLE
Moving Forward

### DISCUSSION THREAD

**Discussion by JoshStrobl** [BuddiesOfBudgie/budgie-desktop#3] [Announcements]
This is sort of a meta post on the direction moving forward. This isn't a specific priority list, just immediate items I'd like to get addressed.

- [Making Budgie 10 Fly Again](https://github.com/BuddiesOfBudgie/budgie-desktop/discussions/2) - Unlocking Budgie 10 development.
- Make sure people are properly enabled in our [Transifex](https://www.transifex.com/buddiesofbudgie/).
- Buddies of Budgie website. Have some mockups of a prior site design in Figma, going to apply it here. Will be for both the site and documentation. Cross-distro documentation, leave no Budgie users behind. Will be written in GatsbyJS, leverage GitHub Actions, etc. Will replace any GitHub wiki.
- Start active discussions on wants / needs for Budgie 11 once Budgie 10 refinement is in full swing. I'd like to heavily experiment with Wayland being the priority, possibly building a wlroots-based compositor / WM.

**Comment by comradekingu** [comment]
@JoshStrobi I think translations on Weblate and donations on Liberapay makes sense.

**Reply by JoshStrobl** [reply]
We already are using Transifex for translations and no plans on supporting Liberapay. OpenCollective handles all the tax-related stuff for us, which is the biggest reason for going with that.
